### DISCUSSION TITLE
How to change source? 如何换源？

### DISCUSSION THREAD

**Discussion by ccmywish** [pdm-project/pdm#2921] [Q&A] [answered]
https://github.com/RubyMetric/chsrc/issues/19

[chsrc](https://github.com/RubyMetric/chsrc) is a command line tool to change source for various kinds of programming languages, OS packages, and some softwares repos. I'm adding feature to support `pdm`. 

`chsrc` 是一个命令行换源工具，用来切换各种编程语言、操作系统包、一些软件仓库的源，对大陆用户非常有用。我想要为`pdm`添加支持。

I'm here to ask if this code piece is enough?
我想询问，下述命令是否是正确的且足够的？
```
# Check present mirror url
pdm config --global pypi.url 

# Change mirror url
pdm config --global pypi.url https://example.com/a/b/c
```

**Comment by frostming** [comment, marked as answer]
`--global` is not needed.
